src/main/java/spotify/api/impl/BrowseApiRetrofit.java

```diff
@@ -83,6 +89,34 @@ public PlaylistSimplifiedPaging getCategoryPlaylists(String categoryId, Map<Stri
         }
     }
 
+    @Override
+    public Paging<PlaylistSimplified> getCategoryPlaylistsUnwrapped(String categoryId, Map<String, String> options) {
```

Comment:
This method is a duplication of https://github.com/jzheng2017/spotify-web-api-wrapper/blob/main/src/main/java/spotify/api/impl/BrowseApiRetrofit.java#L64 with the only difference being the `getPlaylists` call. You can also call the existing function and call `getPlaylists` on that.

Reply:
I was unsure if calling soon to be deprecated methods was a good idea, hence the duplication. Thanks for clearing this up, the new methods call the old methods now.

Reply:
We can call the deprecated method to avoid duplication and when we phase out the deprecated method we can take over its implementation